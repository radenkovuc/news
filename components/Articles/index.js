import ArticleCard from '../ArticleCard'
import styled from 'styled-components'
import React from 'react'

const Container = styled.div`
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
`

type Props = {
  articles: Array
}

const Articles = (props: Props) => {
  const {articles} = props

  return (
    <Container>
      {articles.map((article, index) => {
        return <ArticleCard key={index} article={article} useMobileStyle />
      })}
    </Container>
  )
}

Articles.defaultProps = {
  articles: []
}

export default Articles
