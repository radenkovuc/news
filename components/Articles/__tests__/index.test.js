import React from 'react'
import renderer from 'react-test-renderer'
import Articles from '../'

describe('<Articles />', () => {
  it('match snapshot', () => {
    const tree = renderer.create(<Articles />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
