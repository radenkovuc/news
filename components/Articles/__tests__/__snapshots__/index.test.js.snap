// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Articles /> match snapshot 1`] = `
<div
  className="Container-pznew3-0 bIbXSN"
/>
`;
